src/config/util.ts

```diff
@@ -127,7 +127,7 @@ export const getSushiswapPrice = async (contract: string): Promise<TokenPrice> =
   const sushiswapGraphqlClient = new GraphQLClient(SUSHISWAP_URL);
   const sushiswapGraphqlSdk = getSushiswapSdk(sushiswapGraphqlClient);
   const { pair } = await sushiswapGraphqlSdk.SushiswapPair({
-    id: contract,
+    id: contract.toLowerCase(),
```

Comment:
I'd move this within the `*Pair` methods so that anyone else calling it doesn't care about what they pass in.

Reply:
these are methods from codegen as templates so they have no be given proper input afaik

Reply:
i would wrapper them such that the input is provided as a contract so that this isn't an issue in the future.

this wrapper could also handle things like dealing with errors.